### ISSUE TITLE
Modifiers from being Puppeted

### ISSUE THREAD

**Issue by GrantSP** [Thalassicus/cep-bnw#42]
I'm doing more testing into this puppet bug and I would like some clarification on a couple of issues I have come across.
I'll split them into different issues for easier management.

I started a new game and found the Aztec capital close by and decided to take it before they grow too large. Once taken, and correctly given the options of takeover, I examined the yields this city produces.
There are some "Modifier for Puppet" penalties that I am not sure are supposed to work the way they are shown.

Culture          -25%
Science        -50% & -25% & -50%
Gold             -25% & -25%

There are no penalties to Food and Production.
Is this correct? Are Culture,Science & Gold the only yields penalized?
Also what is with the multiple penalties against Science & Gold? Surely this an error?

I have only just examined it in game and haven't looked through the code yet.

**Comment by GrantSP** [comment]
Must have something to do with the new YieldLibrary. There a lot of deprecated vanilla functions that are now handled by some new modded functions and I am still trying to get my head around them.
I can see that the Global Defines for puppet penalties are changed to show -50% to Science instead of -25 and Faith & Gold now have a penalty of -25% instead of no penalty. Not sure that is a good idea, Faith especially shouldn't be penalized.
Still no clue as to why the multiples occur, and to clarify, I am pretty sure they are totalling as to the yield displayed.
So it is Science -50% then -25% and another -50% off. Or 16 Science becomes 8 then 6 then 3. Makes for a drastic hit on puppet cities.

**Comment by Thalassicus** [comment]
I now set it to 50% science and 0% everything else. I don't know why the modifiers might be showing up multiple times.

I strive to balance military vs peaceful players on the battlefield. It makes sense for military players to have more experienced units. How can peaceful players counter this? More technologically advanced units seem logical. Civ 4 did this with war weariness. We had to divert resources from science into happiness and gold to keep a war going.

The way I do it in Civ 5 is to give peaceful players research advantages wherever possible (like research agreements), and limit warmonger science. Libraries get sacked when a city's captured, puppets focus on gold, and create less science from population.
